frontend/src/router.jsx

```diff
@@ -1,31 +1,26 @@
 import React from 'react';
 import { Switch, Route, Router } from 'react-router-dom';
-import { createBrowserHistory } from 'history';
+import { createBrowserHistory } from 'history'; // eslint-disable-line import/no-extraneous-dependencies
 
 // Pages
 import { Splash } from './pages/Splash';
 
 // The app's history object
 const history = createBrowserHistory();
 
-const RouterComponent = (props) => {
-    return(
-        <Router history={history}>
-				<Route
-		          render={({ location }) => (
-		            <React.Fragment>
-		            	<Switch>
-					        <Route exact path="/">
-                                <Splash />
-                            </Route>
-					    </Switch>
-		            </React.Fragment>
-		          )} />
-		</Router>
-    )
-}
+const RouterComponent = () => { // eslint-disable-line arrow-body-style
```

Comment:
Any specific reason why we are disabling this linter rule here?

Reply:
cus its stupid. It seemed to want the return on one line because it was one element, but its a chunk of a react element that would look ugly on one line.